src/App.js

```diff
@@ -1,18 +1,20 @@
 /* eslint-disable */ 
 import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import Navbar from './components/Navbar';
-import Bookstore from './components/Bookstore';
+import Books from './components/Book';
```

Comment:
This error prevents your app from running correctly. Kindly fix it.

![image](https://user-images.githubusercontent.com/27709832/203927271-42b1557f-7692-4ba3-a82e-23eaf9af1c4d.png)